src/views/Assignments/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { useMemo, useState } from 'react'
+import { ViewHeader } from '@/components'
+import { BriefcaseBusinessIcon } from '@ttab/elephant-ui/icons'
+import { ScrollArea, Tabs, TabsContent } from '@ttab/elephant-ui'
+import { TableProvider } from '@/contexts/TableProvider'
+import { TableCommandMenu } from '@/components/Commands/TableCommand'
+import { Header } from '@/components/Header'
+import { AssignmentsList } from './AssignmentsList'
+import { assignmentColumns } from './AssignmentColumns'
+import { Commands } from '@/components/Commands'
+import { useAuthors } from '@/hooks/useAuthors'
+import { useRegistry } from '@/hooks/useRegistry'
+import { useSession } from 'next-auth/react'
+import { type ViewMetadata } from '@/types'
+import { type AssignmentMeta } from './types'
+import { type IDBAuthor } from 'src/datastore/types'
+import { useQuery } from '@/hooks/useQuery'
+import { getDateTimeBoundariesUTC } from '@/lib/datetime'
+
+const meta: ViewMetadata = {
+  name: 'Assignments',
+  path: `${import.meta.env.BASE_URL}/assignments`,
+  widths: {
+    sm: 12,
+    md: 12,
+    lg: 6,
+    xl: 6,
+    '2xl': 6,
+    hd: 6,
+    fhd: 4,
+    qhd: 3,
+    uhd: 2
+  }
+}
+
+export const Assignments = (): JSX.Element => {
+  const query = useQuery()
+  const { from } = useMemo(() =>
+    getDateTimeBoundariesUTC(query.from
+      ? new Date(`${query.from}T00:00:00.000Z`)
+      : new Date()),
+  [query.from])
+  const [currentTab, setCurrentTab] = useState<string>('list')
+  const authors = useAuthors()
+  const { locale, timeZone } = useRegistry()
+  const { data: session } = useSession()
+
+  const assigneeUserName = useMemo(() => {
+    const userEmail = session?.user.email
+    const author = authors.find((_: IDBAuthor) => _?.email === userEmail)
```

Comment:
Might be better to use a instead of _ as Gustav just made me aware that it is common to use _ for parameters that aren't used (typescript doesn't even complain on unused parameters when using it).

Reply:
Pushed new commit to fix this.